### PR TITLE
add validation for identity extensions

### PR DESCRIPTION
**Pull request by dgrossen**
## Description

Add a validation to check if the Identity Extension for Experience Cloud and the Identity Extension for Edge Network have been installed and registered.

## Related Issue

Help customers identify which Identity extensions need to be registered.

https://jira.corp.adobe.com/browse/MOB-17818?focusedCommentId=34050547&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-34050547

https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network/faq/#q-i-am-using-aep-edge-and-adobe-solutions-extensions-which-identity-extension-should-i-install-and-register

## Motivation and Context

Recently, a customer wasn't getting Analytics Response events in Assurance Tools and we discovered that while they had Identity configured for Edge Network they did not have the core Identity extension for Experience Cloud solutions like Analytics.

## How Has This Been Tested?

Unit tests and in the validation editor

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: plugins/identity-installed/index.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  const identity = extensions['com.adobe.module.identity'];
+  const edgeIdentity = extensions['com.adobe.edge.identity'];
+  const edge = extensions['com.adobe.edge'];
+
+  const solution =
+    extensions['com.adobe.module.analytics'] ||
+    extensions['com.adobe.module.target'] ||
+    extensions['com.adobe.module.places'] ||
+    extensions['com.adobe.module.campaign'];
```
Comment (emdobrin) [inline plugins/identity-installed/index.ts:35]: let's also add `com.adobe.module.media` in the mix

File: plugins/identity-installed/plugin.test.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import {
```
Comment (emdobrin) [inline plugins/identity-installed/plugin.test.ts:1]: missing ©️

**Review by emdobrin — COMMENTED** [review]
Looks great @dgrossen thanks for this. Can we get this enabled for the Analytics and Event transaction views to begin with?

**Review by longAdobe — APPROVED** [review]
👍

**Pull request merged**